updater/lib/dependabot/updater/group_update_creation.rb

```diff
@@ -241,6 +241,9 @@ def compile_updates_for(dependency, dependency_files, group) # rubocop:disable M
           return []
         end
 
+        # Raise an error if the package manager version is unsupported
+        dependency_snapshot.package_manager&.raise_if_unsupported!
```

Comment:
🔎 Is the `&.` needed here for changes which happen across ecosystems?

Reply:
Yes, because currently other eco-system don't have package_manager implementation. We always need package manager object from eco-system to be set to do this operations.